router/sources/router.move

```diff
@@ -437,7 +443,7 @@ module router::router {
 
     // ==== EXPIRATION ====
 
-    // Not available in MODE_V1
+    /// @notice Renews the domain. NOTE 1: Not available in MODE_V1. NOTE 2: Will attempt to migrate the name if applicable
```

Comment:
NOTE 2: Will attempt to migrate the name if applicable and will fail if it's subdomain, please to call `migrate_name` first